### ISSUE TITLE
[Bug] GuidanceBackend only fast tokenizers are supported

### ISSUE THREAD

**Issue by Prayer3th** [sgl-project/sglang-jax#464]
### Checklist

- [x] 1. I have searched related issues but cannot get the expected help.
- [x] 2. The bug has not been fixed in the latest version.
- [x] 3. Please note that if the bug-related issue you submitted lacks corresponding environment info and a minimal reproducible demo, it will be challenging for us to reproduce and resolve the issue, reducing the likelihood of receiving feedback.
- [x] 4. If the issue you raised is not a bug but a question, please raise a discussion at https://github.com/sgl-project/sgl-jax/discussions/new/choose Otherwise, it will be closed.
- [x] 5. Please use English, otherwise it will be closed.

### Describe the bug

<img width="3840" height="1328" alt="Image" src="https://github.com/user-attachments/assets/372baac9-c5a1-4cd0-8780-1d3bc86916b0" />

### Reproduction

JAX_COMPILATION_CACHE_DIR=/tmp/jit_cache  python3 -u -m sgl_jax.launch_server --model-path /dev/shm/models--Qwen--Qwen-72B --trust-remote-code  --dist-init-addr=10.128.0.90:10011 --nnodes=4  --tp-size=16 --ep-size=16 --device=tpu --random-seed=3 --mem-fraction-static=0.85 --chunked-prefill-size=2048 --download-dir=/dev/shm --dtype=bfloat16 --max-running-requests 256 --skip-server-warmup --page-size=128 --node-rank=0

### Environment

v6e-16
